pennylane_lightning/core/src/simulators/lightning_gpu/CMakeLists.txt

```diff
@@ -27,11 +27,14 @@ if(ENABLE_MPI)
     list(APPEND LGPU_FILES StateVectorCudaMPI.cpp)
 endif()
 
-
 add_library(${PL_BACKEND} STATIC ${LGPU_FILES})
 
 target_compile_options(lightning_compile_options INTERFACE "-D_ENABLE_PLGPU=1")
 
+if(ENABLE_MPI)
+    target_compile_options(lightning_compile_options INTERFACE "-D_ENABLE_PLGPU_MPI=1")
```

Comment:
I assume we don't also need to enable the single-GPU definition here too?

Reply:
we need and `ENABLE_PLGPU` is enabled at line 32, just above  `if(ENABLE_MPI)` at line 34.